core/server/master/src/main/java/alluxio/master/AlluxioSimpleMasterProcess.java

```diff
@@ -73,22 +73,27 @@ public abstract class AlluxioSimpleMasterProcess extends MasterProcess {
 
   @Override
   public void start() throws Exception {
+    LOG.info("Start AlluxioSimpleMasterProcess");
+    // RPC service and web service here
     mServices.forEach(SimpleService::start);
     mJournalSystem.start();
     // the leader selector is created in state STANDBY. Once mLeaderSelector.start is called, it
     // can transition to PRIMARY at any point.
     mLeaderSelector.start(getRpcAddress());
 
     while (!Thread.interrupted()) {
+      startMasterComponents(false);
```

Comment:
this is needed, because standby job master process now needs to start its JobMaster

Reply:
can you add a comment and move line 101 to the beginning of the while loop